optaplanner-examples/src/main/java/org/optaplanner/examples/tsp/app/TspApp.java

```diff
@@ -55,7 +55,7 @@ protected TspPanel createSolutionPanel() {
 
     @Override
     public SolutionFileIO<TspSolution> createSolutionFileIO() {
-        return new XStreamSolutionFileIO<>(TspSolution.class);
+        return new TspXStreamSolutionFileIO();
```

Comment:
Why extract that in a separate file? The more files in an example, the harder it is to understand for our users.

Reply:
I thought you were going to ask. :-) I have a harness that allows me to run all examples in a single benchmark - essentially an abstraction to easily perform score director benchmarks over all examples and all score directors, without having a benchmark template of a million lines.

The downside of that is that it relies pretty heavily on generics. And I had to do this to standardize it with the other examples. (This was the only one that didn't have its own solution file IO.)

I will submit the harness as a PR later and we can discuss it.

Reply:
If all other examples do it, there is very little point indeed in this example not doing it to be easier to understand. +1 for this change (consistency)